pages/advanced/cli-reference/common-commands.mdx

```diff
@@ -252,7 +252,7 @@ change_master_copy <address>
 ## Update to L2
 
 <Callout type="warning" emoji="⚠️">
-  A non-L2 Safe can only be migrated to L2 if the non-L2 Safe was not used before (nonce must be zero).
+  A non-L2 Safe can only be migrated to L2 if the L2 Safe was not used before (nonce must be zero).
```

Comment:
```suggestion
  A non-L2 Safe can only be migrated to L2 if the non-L2 Safe was not used before (nonce must be zero).
```

Reply:
@Uxio0 is this statement correct?
`A non-L2 Safe can only be migrated to L2 if the non-L2 Safe was not used before (nonce must be zero).`

I have recently claimed multisig on Base mainnet that has nonce 0 on Base  and nonce greater 0 on Ethereum mainnet

Ethereum mainnet can be migrated if Base nonce is 0 otherwise Gnosis Safe UI wont index and Safe wont load in UI.

Nonce must be 0 on L2 before migration is done. no? If I understand correct statement should says. 
`A non-L2 Safe can only be migrated to L2 if the L2 Safe was not used before (nonce must be zero).`

Reply:
A non-L2 Safe can be deployed in any network, and that's what's the contract is for, to migrate to the L2 version. It has nothing to do with the chain

Reply:
Got you. Sorry for confusion.